tests/tests.schema

```diff
@@ -3864,6 +3923,63 @@
         },
         {
           "$ref": "#/definitions/Microsoft.RegexRecognizer"
+        },
+        {
+          "$ref": "#/definitions/Microsoft.AgeEntityRecognizer"
+        },
+        {
```

Comment:
Hmm.  These are supposed to be alphabetical.  This might be a bug in dialog:merge.